makefile.shade

```diff
@@ -70,7 +71,67 @@ var buildTarget = "compile"
 
 #verify-all .pull .change-default-build-target-to-verify .build-all
 
-#build-and-install-all .change-default-build-target-for-coherence-build .build-all
+#ci-build
+    @{
+        var ciVolatileShare = Environment.GetEnvironmentVariable("CI_VOLATILE_SHARE");
+        var nugetExe = Environment.GetEnvironmentVariable("PUSH_NUGET_EXE");
+        var universeArtifacts = Path.Combine("artifacts", "build");
+        Directory.CreateDirectory(universeArtifacts);
+        buildTarget = Environment.GetEnvironmentVariable("KOREBUILD_BUILD_TARGETS") ?? "--quiet compile nuget-install";
+
+        foreach (var batch in BatchReporties())
+        {
+            Parallel.ForEach(batch, repoInfo =>
```

Comment:
Reporting will suck here... Can we also drop a file/repo with that repo's output so a human can read it?